### PR TITLE
Features/uintra2 16 move groups

### PR DESCRIPTION
**Pull request by svinnichenko**

### CODE REVIEW

File: src/Uintra20/Content/uintra20/src/app/feature/project/specific/groups/group-details-wrapper/group-details-wrapper.component.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+          {{data.title.title}}
+        </a>
+      </h2>
+      <ul *ngIf="data.groupLinks" class="tabs col-md-8">
+        <li *ngFor="let tab of data.groupLinks" class="tabs__item">
+          <a
+            [routerLink]="tab.link.originalUrl | ulink"
+            class="tabs__item-link"
+            routerLinkActive="active"
+            [routerLinkActiveOptions]="{ exact: true }">
```
Comment (razrabotal) [inline src/Uintra20/Content/uintra20/src/app/feature/project/specific/groups/group-details-wrapper/group-details-wrapper.component.html]: It makes sense? Subpages should be have an active class in the current tab

File: src/Uintra20/Content/uintra20/src/app/feature/project/specific/groups/group-details-wrapper/group-details-wrapper.component.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+            {{tab.title}}
+          </a>
+        </li>
+        <li *ngIf="data.groupEditPageLink" class="tabs__item tabs__item--settings">
+          <a
+            [routerLink]="data.groupEditPageLink.link.originalUrl | ulink"
+            class="tabs__item-link tabs__item-link--settings"
+            routerLinkActive="active"
+            [routerLinkActiveOptions]="{ exact: true }">
```
Comment (razrabotal) [inline src/Uintra20/Content/uintra20/src/app/feature/project/specific/groups/group-details-wrapper/group-details-wrapper.component.html]: It makes sense? Subpages should be have an active class in the current tab

File: src/Uintra20/Content/uintra20/src/app/feature/project/specific/groups/group-details-wrapper/group-details-wrapper.component.less [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    margin-bottom: 15px;
+
+    &-link {
+      color: @heading-bg;
+      text-decoration: none;
+
+      &:hover {
+        text-decoration: underline;
+      }
+    }
```
Comment (razrabotal) [inline src/Uintra20/Content/uintra20/src/app/feature/project/specific/groups/group-details-wrapper/group-details-wrapper.component.less]: Maybe use a mixin

File: src/Uintra20/Content/uintra20/src/app/feature/project/specific/groups/group-details-wrapper/group-details-wrapper.component.less [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+  position: relative;
+  font-size: 13px;
+  line-height: 15px;
+  font-weight: bold;
+  padding-left: 0;
+  padding-right: 0;
+
+  @media @md-up {
+    display: flex;
+  }
```
Comment (razrabotal) [inline src/Uintra20/Content/uintra20/src/app/feature/project/specific/groups/group-details-wrapper/group-details-wrapper.component.less]:
```
 display: flex;

@media @md-down {
    display: none;
}
```

File: src/Uintra20/Content/uintra20/src/app/feature/project/specific/groups/groups-form/groups-form.component.html [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+    <h2 class="groups-create__row groups-create__title">{{edit ? 'Edit group settings' : 'Create Group'}}</h2>
+
+    <div class="groups-create__row">
+      <h3 class="subtitle required">Group title</h3>
+      <text-input
+        type="text"
+        aria-label="Title"
+        [maxLength]="TITLE_MAX_LENGTH"
+        class="groups-create__title-input"
+        [ngClass]="{'invalid': isShowValidation && !title || isShowValidation && title.length > 200}"
```
Comment (razrabotal) [inline src/Uintra20/Content/uintra20/src/app/feature/project/specific/groups/groups-form/groups-form.component.html]: move the logic from view to .ts file

File: src/Uintra20/Content/uintra20/src/app/feature/project/specific/groups/groups-form/groups-form.component.less [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+      display: flex;
+      align-items: center;
+      justify-content: center;
+    }
+
+    img {
+      width: 100%;
+    }
+  }
+}
```
Comment (razrabotal) [inline src/Uintra20/Content/uintra20/src/app/feature/project/specific/groups/groups-form/groups-form.component.less]: Maybe move to global styles or component

File: src/Uintra20/Content/uintra20/src/app/feature/project/specific/groups/groups-form/groups-form.component.ts [hunk 7]
```diff
@@ -0,0 +1,99 @@
...
+import { MAX_FILES_FOR_SINGLE } from 'src/app/constants/dropzone/drop-zone.const';
+
+@Component({
+  selector: 'groups-form',
+  templateUrl: './groups-form.component.html',
+  styleUrls: ['./groups-form.component.less']
+})
+export class GroupsFormComponent {
+  @Input() data: any;
+  @Input('edit') edit: boolean;
```
Comment (razrabotal) [inline src/Uintra20/Content/uintra20/src/app/feature/project/specific/groups/groups-form/groups-form.component.ts]: try to use attribute
 [https://angular.io/api/core/Attribute](https://angular.io/api/core/Attribute) 
[https://netbasal.com/getting-to-know-the-attribute-decorator-in-angular-4f7c9fb61243](https://netbasal.com/getting-to-know-the-attribute-decorator-in-angular-4f7c9fb61243)

File: src/Uintra20/Content/uintra20/src/app/feature/project/specific/groups/group-details-wrapper/group-details-wrapper.component.html (hunk 1 above)
Comment (eugene-sahach) [reply src/Uintra20/Content/uintra20/src/app/feature/project/specific/groups/group-details-wrapper/group-details-wrapper.component.html]: +

File: src/Uintra20/Content/uintra20/src/app/feature/project/specific/groups/group-details-wrapper/group-details-wrapper.component.html (hunk 2 above)
Comment (eugene-sahach) [reply src/Uintra20/Content/uintra20/src/app/feature/project/specific/groups/group-details-wrapper/group-details-wrapper.component.html]: +

File: src/Uintra20/Content/uintra20/src/app/feature/project/specific/groups/group-details-wrapper/group-details-wrapper.component.less (hunk 4 above)
Comment (eugene-sahach) [reply src/Uintra20/Content/uintra20/src/app/feature/project/specific/groups/group-details-wrapper/group-details-wrapper.component.less]: +

File: src/Uintra20/Content/uintra20/src/app/feature/project/specific/groups/group-details-wrapper/group-details-wrapper.component.less (hunk 3 above)
Comment (eugene-sahach) [reply src/Uintra20/Content/uintra20/src/app/feature/project/specific/groups/group-details-wrapper/group-details-wrapper.component.less]: +

File: src/Uintra20/Content/uintra20/src/app/feature/project/specific/groups/groups-form/groups-form.component.html (hunk 5 above)
Comment (eugene-sahach) [reply src/Uintra20/Content/uintra20/src/app/feature/project/specific/groups/groups-form/groups-form.component.html]: +

File: src/Uintra20/Content/uintra20/src/app/feature/project/specific/groups/groups-form/groups-form.component.less (hunk 6 above)
Comment (eugene-sahach) [reply src/Uintra20/Content/uintra20/src/app/feature/project/specific/groups/groups-form/groups-form.component.less]: +

**Pull request closed without merging**